tools/amGithubNotifier/README.md

```diff
@@ -0,0 +1,32 @@
+# amGithubNotifier - Alertmanager GitHub Webhook Receiver
+
+A simple bridge for receiving Alertmanager alerts and posting comments to github.
+
+It listens at the `/hook` endpoint on port `:8080` by default.
+
+## Usage
+
+> Note: All alerts sent to amGithubNotifier must have the `prNum` label and `description` annotation, `owner` and `repo` labels are optional.
+
+Example `alerts.rules.yml`:
```

Comment:
can you put this example also in the main.go and print it with the -h flag.

Reply:
I think it's not possible to alter the helptext in kingpin yet, will have to add another flag for this then?

I added the example in`main.go`